ivy-data/src/main/java/com/ivy/data/db/dao/write/WriteTagTransactionDao.kt

```diff
@@ -0,0 +1,29 @@
+package com.ivy.data.db.dao.write
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.ivy.data.db.entity.TagTransactionEntity
+import java.util.*
+
+@Dao
+interface WriteTagTransactionDao {
```

Comment:
Can we rename it to `TransactionTagEntity` and all corresponding classes  it sounds nicer because it represents a tag that's attached to a transaction.

Reply:
Ig we should rename it to something generic I was planning to associate tags with other "Entities" also eg. Loans 
Something like 
WriteTagAssociationDao

Reply:
Got it